distribution/sender/routes/active-mq-publisher.xml

```diff
@@ -0,0 +1,132 @@
+<routes xmlns="http://camel.apache.org/schema/spring">
+
+    <route id="active-mq-publisher" errorHandlerRef="outBoundErrorHandler">
```

Comment:
I'm thinking the error handler for this route or the out-bound-db-sync should be different otherwise if an error occurs in this route, the retry mechanism might fail or might take it through the previous routes which you might not want

Reply:
We need to think about this carefully otherwise we might end up in circles in case of errors, I will think through it, did you actually test the functionality including if an error occurs? And see what happens?

Reply:
Please use the **shutdownErrorHandler** here otherwise the **outBoundErrorHandler** won't work when it is invoked in case of an exception in this route

Reply:
Did you see these comments?

Reply:
I've changed to the **shutdownErrorHandler** as you suggested. Thanks.